TESTS/network/emac/emac_TestMemoryManager.cpp

```diff
@@ -43,12 +48,57 @@
 
 char s_trace_buffer[100] = MEM_MNGR_TRACE;
 
+/* For LPC boards define the heap memory bank ourselves to give us section placement
+   control */
+#ifndef ETHMEM_SECTION
```

Comment:
If we ever do decide to support Nanostack on this board, let's figure out how to put this somewhere central. But this'll do for test.

Reply:
Yes, that is true.